security/clamav/pkg-descr

```diff
@@ -9,6 +9,10 @@ database updates.
 Plugin Changelog
 ================
 
+1.8.1
+
+* Fixed detect broken executables option
```

Comment:
You can add your (conttributed by _GITHUBHANDLE_) here if you want. If a changelog exists it is linked from the release notes instead of giving credit there.